curriculum/js1/8.js

```diff
@@ -5,8 +5,18 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, c=a) => {
+  if (c<=1) {
+    return c;
+  }
+  if ((a%c)===0) {
+    if ((b%c)===0) {
```

Comment:
How could you combine these conditions so that one `if` statement could replace the `if` statements on lines 12 & 13?